lib/msf/core/db_manager/module_cache.rb

```diff
@@ -212,15 +211,6 @@ def search_modules(search_string)
         formatted_values = match_values(value_set)
 
         case keyword
-          when 'app'
```

Comment:
This block appeared to be the only portion that was used to assign values for `@stances`, should the related `@stances` code in this method be removed as well?

Reply:
Modules do still have a stance,

```
res[:stance] = m.stance.to_s.index("aggressive") ? "aggressive" : "passive"
```

maybe we need to relink that somewhere?

Reply:
Looks like `@stances` though can completely remove, updating